alphabet-Order-task.js

```diff
@@ -0,0 +1,11 @@
+// Napisz funkcję alphabetOrder(), która zwróci podany łańcuch znaków z literami
+// uporządkowanymi według alfabetu („wyraz” - „arwyz”). Nie przejmuj się na razie polskimi
+// znakami :)
+
+function alphabetOrder(string) {
+    return string.split("").sort().join("");
+
+}
+
+console.log(alphabetOrder("dupa")); // adpu
+console.log(alphabetOrder("placki ziemniaczane")); //  aaacceeiiiklmnnpzz
```

Comment:
It's not explicit. You have to show somehow that this space is there. That these two spaces aren't there by an accident.